packages/styled-components/src/utils/test/generateAlphabeticName.test.js

```diff
@@ -18,4 +18,18 @@ describe('generateAlphabeticName', () => {
     expect(generateAlphabeticName(2733)).toMatchInlineSnapshot(`"a-D"`);
     expect(generateAlphabeticName(7390035)).toMatchInlineSnapshot(`"a-Da-d"`);
   });
+
+  describe('with SC_CLASS_PREFIX', () => {
+    beforeEach(() => {
+      jest.resetModules();
+    });
+
+    it.only('should concat SC_CLASS_PREFIX with generated alphabetic names', () => {
```

Comment:
I guess this is a development leftover, `only`.
👀

Reply:
Oh, my bad, thanks for the review :smile: